### ISSUE TITLE
luci-app-dockerman: sub menu not showing

### ISSUE THREAD

**Issue by ysc3839** [openwrt/luci#6168]
## Steps to reproduce:
1. Install `luci-app-dockerman`.
2. Reload luci page.

## Actual behavior:
The `Docker` menu contains no sub menu.
![image](https://user-images.githubusercontent.com/12028138/209262338-f0bfa920-3bb2-401b-ace6-8cf68dda9e2e.png)

## Expected behavior:
The `Docker` menu should contains sub menu.
![image](https://user-images.githubusercontent.com/12028138/209262431-5432762d-c406-423e-b85a-5e5cc81ea55d.png)

## Additional Information:
Replace `alias("admin", "docker", "config")` with `firstchild()` resolve this issue.
```diff
--- a/applications/luci-app-dockerman/luasrc/controller/dockerman.lua
+++ b/applications/luci-app-dockerman/luasrc/controller/dockerman.lua
@@ -9,7 +9,7 @@ module("luci.controller.dockerman",package.seeall)

 function index()
        entry({"admin", "docker"},
-               alias("admin", "docker", "config"),
+               firstchild(),
                _("Docker"),
                40).acl_depends = { "luci-app-dockerman" }
```
IIRC there's no such issue in previous version of luci. Is this expected behavior?

OpenWrt version information from system `/etc/openwrt_release`
```
DISTRIB_ID='OpenWrt'
DISTRIB_RELEASE='22.03.2'
DISTRIB_REVISION='r19803-9a599fee93'
DISTRIB_TARGET='x86/64'
DISTRIB_ARCH='x86_64'
DISTRIB_DESCRIPTION='OpenWrt 22.03.2 r19803-9a599fee93'
DISTRIB_TAINTS=''
```

**Comment by dannygonzalez0861** [comment]
I just updated OPKG and the same issue has now happened to me. My dockers are running as i can still access them but I too have lost the sub menu's as yuo have:
![image](https://user-images.githubusercontent.com/61332341/209866001-1e1720da-92de-4398-a44b-86c28810c1a0.png)

**Comment by JKSTAFF** [comment]
I make a snapshot image with dockerman integrated. Droplist menu lost after reconfig docker root path or create some containers. Luci unable to dispatch.

**Comment by kofec** [comment]
I confirm the same issue on latest 22.03 but in 21.02 all is OK

**Comment by kofec** [comment]
Hi @lisaac https://github.com/lisaac/luci-app-dockerman - maybe we should report here ?

**Comment by kofec** [comment]
@ysc3839 Thanks - your workaround/solution is working fine

**Comment by hnyman** [comment]
luci-app-dockerman in master was apparently broken and then fixed in April with commit 0c1fc7f6 from PR #5789
Package in 22.03 was fine at that time.
22.03 has then got broken in Dec 2022 due to backport commit fc759abea 

I will backport the dockerman fix 0c1fc7f6  from master.

**Comment by hnyman** [comment]
Shoudl be fixed with 68e23b284
